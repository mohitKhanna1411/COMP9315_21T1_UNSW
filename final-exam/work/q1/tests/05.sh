./pages 5 6 1,1,2,2,3,3 <<_Q_
a,b,c,d,e
a,b,?,?,?
?,b,c,d,e
a,?,c,d,e
a,b,?,d,e
_Q_
