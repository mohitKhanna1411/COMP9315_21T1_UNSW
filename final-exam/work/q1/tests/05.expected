CV = < (1,0) (1,1) (2,0) (2,1) (3,0) (3,1) >
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = 11101011 01100101 01010001 10111010
q[3] = 01101100 01000111 11011011 00100111
q[4] = 00110110 00001101 11011001 10101111
q[5] = 01111010 00000000 10101001 00001100
Pages: 57
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = 11101011 01100101 01010001 10111010
q[3] = ??
q[4] = ??
q[5] = ??
Pages: 9 25 41 57
query> q[1] = ??
q[2] = 11101011 01100101 01010001 10111010
q[3] = 01101100 01000111 11011011 00100111
q[4] = 00110110 00001101 11011001 10101111
q[5] = 01111010 00000000 10101001 00001100
Pages: 56 57 58 59
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = ??
q[3] = 01101100 01000111 11011011 00100111
q[4] = 00110110 00001101 11011001 10101111
q[5] = 01111010 00000000 10101001 00001100
Pages: 49 53 57 61
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = 11101011 01100101 01010001 10111010
q[3] = ??
q[4] = 00110110 00001101 11011001 10101111
q[5] = 01111010 00000000 10101001 00001100
Pages: 9 25 41 57
query> 
